lib/convertor.js

```diff
@@ -47,10 +48,9 @@ class MongoToKnex {
      * Determine if statement lives on parent table or if statement refers to a relation.
      */
     processStatement(column, op, value) {
-        const columnParts = column.split('.');
-
+        const [tableName, columnName] = column.split('.');
```

Comment:
Improved the naming a bit as it was hard to see right away what columnParts[0] and columnParts[1] actually mean

Reply:
Same goes for all the stuff that's just rejigging tests on its own without any related code changes.